rir/src/compiler/translations/pir_2_rir/pir_2_rir.cpp

```diff
@@ -1336,8 +934,8 @@ rir::Code* Pir2Rir::compileCode(Context& ctx, Code* code) {
     });
     cb.flush();
 
-    auto localsCnt = alloc.slots();
-    auto res = ctx.finalizeCode(localsCnt);
+    auto res =
```

Comment:
I feel like some documentation get's lost with this change. why are we passing slots() ? in the old version we knew: because its the locals count...